Snakefile-sorting-hat

```diff
@@ -91,6 +91,10 @@ def get_outputs():
         valid_acc_dir = next(os.walk( config['atlas_exps'] ))[1]
         # intersection of the two lists
         pass_ACC = list(set(ACC) & set(valid_acc_dir))
+        #save passing ACCs to file
+        with open('exps_tagged_for_processing.txt', 'w') as f:
```

Comment:
looks good, but maybe you want to set the file path here from config, otherwise it will end up in the analysis directory from where it is getting experiments.